vignettes/model-card.Rmd

```diff
@@ -0,0 +1,76 @@
+    
+ 
+If you are not using RStudio, you'll also need to install [Pandoc](https://pandoc.org). Then, use the `rmarkdown::draft()` function to create the model card:
+
+```r
+rmarkdown::draft(
+    "my_model_card.Rmd", template = "vetiver_model_card", package = "vetiver"
+)
+```
+
+## Model card outline
+
+There are several basic sections in the model card framework used here.
+
+- **Model details:** Some details about your model can be determined from the model object itself, but some (like who developed the model and license or citation information) need to be provided by you.
+
+- **Intended use:** Outline the intended use and users of the model, and perhaps also what types of use would be out of scope.
+
+- **Important aspects/factors:** What are the demographic, environmental, technical or other aspects that are relevant to the context of the model?
+
+- **Metrics:** Communicate which metrics are being used to evaluate the model, and why these are a good fit for the model's context and domain.
+
+- **Training data & evaluation data:** Some specific dataset was used to train the model, so be sure to share basic details about its characteristics. (Some information about the training data can be extracted from the model object itself.) Some (probably different) specific dataset is used to _evaluate_ the model in the context of the model card, so also explain what the evaluation data is like.
+
+- **Quantitative analyses:** Provide the results of evaluating the model using your chosen metrics and the evaluation data. Be sure to present both overall results (for the dataset as a whole) and disaggregated results, especially with any aspects (demographic, environmental, or other) in mind that have been identified as important for this model. You can use both tables and visualization to present these quantitative analyses.
+
+- **Ethical considerations:** Share ethical considerations and any possible solutions considered. Some specific aspects to consider are sensitive data used, impact on human life, possible risks and harms, and important use cases.
+
+- **Caveats & recommendations:** As the model developer, you likely have the most domain knowledge of what the model can and cannot do, and this section is a good place to share any additional thoughts.
+
+## Can't I just delete the section on ethical considerations?
+
+It's possible that a given machine learning model may not have obvious caveats, ethical challenges, or demographic aspects, or that they are largely unknown. However, we strongly advise that instead of deleting any such section because you have incomplete or imprecise information, you note your own process and considerations. The process of documenting the extent and limits of a machine learning system is part of transparent, responsible reporting. A model card framework such as this is a helpful tool and some parts of a model card can be automated, but ultimately the extent of its value depends on you. From Mitchell et al. (2019):
```

Comment:
Maybe at some point in this paragraph - or already right in the introduction - suggest that if no possible problems come to mind, but the thing itself is people-related in any way, one could try getting feedback from minority groups?